raiden-dapp/src/components/overlays/InfoOverlay.vue

```diff
@@ -38,60 +40,77 @@ export default class InfoOverlay extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '@/scss/dimensions';
 @import '@/scss/mixins';
 @import '@/scss/colors';
 
 .info-overlay {
-  align-items: center;
   background-color: $transfer-screen-bg-color;
   border-radius: 10px;
-  display: flex;
-  flex-direction: column;
-  height: calc(844px);
-  padding: 0 46px;
+  height: 844px;
   position: absolute;
   width: 100%;
   z-index: 102;
+
   @include respond-to(handhelds) {
     border-radius: 0;
     height: 100vh;
+    overflow-y: auto;
   }
 
-  &__header {
+  &__content {
+    align-items: center;
     display: flex;
-    height: 80px;
-    width: 100%;
-  }
+    flex-direction: column;
+    height: 100%;
+    @include respond-to(handhelds) {
+      margin-top: $ios-safe-area;
+    }
 
-  &__close-icon {
-    width: 100%;
-    display: flex;
-    justify-content: flex-end;
-  }
+    &__header {
+      display: flex;
+      height: 80px;
+      width: 100%;
+      @include respond-to(handhelds) {
+        height: 60px;
+      }
+    }
 
-  &__image {
-    margin-top: 46px;
-    height: auto;
-    width: 300px;
-  }
+    &__close-icon {
+      display: flex;
```

Comment:
This does not work. The content div has flex column. Thereby it floats below the header due to the listing. It needs to be `absolute` and then set `right` and `top`.

Reply:
It is not floating below the header because of the flex. It is below the header regardless and by design. The flex is simply there to be able to use `justify-content: flex-end;` to push the x-icon to the far left.

Reply:
But... I don't understand. I mean this can't work. Maybe invite me to a call to discuss this quickly.

Reply:
Okay in the call in turned out that this is the intended design. For me it looked broken. But if that is how it should look like, it is correctly implemented. 👍🏾